### DISCUSSION TITLE
With PIA downloading works perfectly, but uploading doesn't work (0 peers)

### DISCUSSION THREAD

**Discussion by Zjaaspoer** [haugene/docker-transmission-openvpn#2668] [Q&A]
### Is there a pinned issue for this?

- [X] I have read the pinned issues and could not find my issue

### Is there an existing or similar issue/discussion for this?

- [X] I have searched the existing issues
- [X] I have searched the existing discussions

### Is there any comment in the documentation for this?

- [X] I have read the documentation, especially the FAQ and Troubleshooting parts

### Is this related to a provider?

- [X] I have checked the provider repo for issues
- [X] My issue is NOT related to a provider

### Are you using the latest release?

- [X] I am using the latest release

### Have you tried using the dev branch latest?

- [ ] I have tried using dev branch

### Docker run config used

  media_torrent:
    cap_add:
      - NET_ADMIN
    container_name: media_torrent
    dns:
      - 8.8.8.8
    environment:
      LOCAL_NETWORK: 192.168.1.0/24
      OPENVPN_CONFIG: netherlands
      OPENVPN_PASSWORD: ***
      OPENVPN_PROVIDER: PIA
      OPENVPN_USERNAME: ***
      PGID: 1000
      PUID: 1000
      TRANSMISSION_DOWNLOAD_DIR: /downloads
      TRANSMISSION_IDLE_SEEDING_LIMIT_ENABLED: 'false'
      TRANSMISSION_RATIO_LIMIT: 1
      TRANSMISSION_RATIO_LIMIT_ENABLED: 'true'
      TRANSMISSION_SPEED_LIMIT_UP: 1000
      TRANSMISSION_SPEED_LIMIT_UP_ENABLED: 'true'
      TZ: Europe/Amsterdam
    image: haugene/transmission-openvpn:latest
    ports:
      - '8021:9091'
    restart: always
    volumes:
      - media_torrent_downloads:/downloads:rw
      - /mnt/hdd/media/movies:/movies:rw
      - media_torrent_data:/data:rw

### Current Behavior

Downloading without a problem. But peers can't connect to me so I cannot seed.

### Expected Behavior

I would like to be able to upload/seed

### How have you tried to solve the problem?

Tried everything

### Log output

media_torrent    | 2023-07-19T18:44:47.037441651Z Starting container with revision: 1103172c3288b7de681e2fb7f1378314f17f66cf
media_torrent    | 2023-07-19T18:44:47.037873792Z TRANSMISSION_HOME is currently set to: /config/transmission-home
media_torrent    | 2023-07-19T18:44:47.038148599Z WARNING: Deprecated. Found old default transmission-home folder at /data/transmission-home, setting this as TRANSMISSION_HOME. This might break in future versions.
media_torrent    | 2023-07-19T18:44:47.038491399Z We will fallback to this directory as long as the folder exists. Please consider moving it to /config/transmission-home
media_torrent    | 2023-07-19T18:44:47.082163006Z Creating TUN device /dev/net/tun
media_torrent    | 2023-07-19T18:44:47.085335395Z Using OpenVPN provider: PIA
media_torrent    | 2023-07-19T18:44:47.085841276Z Running with VPN_CONFIG_SOURCE auto
media_torrent    | 2023-07-19T18:44:47.086196389Z Provider PIA has a bundled setup script. Defaulting to internal config
media_torrent    | 2023-07-19T18:44:47.088111483Z Executing setup script for PIA
media_torrent    | 2023-07-19T18:44:47.091262520Z Downloading OpenVPN config bundle openvpn into temporary file /tmp/tmp.g0A5XlpuCP
media_torrent    | 2023-07-19T18:44:47.225014315Z Extract OpenVPN config bundle into PIA directory /etc/openvpn/pia
media_torrent    | 2023-07-19T18:44:47.233079294Z Starting OpenVPN using config netherlands.ovpn
media_torrent    | 2023-07-19T18:44:47.234788071Z Modifying /etc/openvpn/pia/netherlands.ovpn for best behaviour in this container
media_torrent    | 2023-07-19T18:44:47.235052628Z Modification: Point auth-user-pass option to the username/password file
media_torrent    | 2023-07-19T18:44:47.235857295Z Modification: Change ca certificate path
media_torrent    | 2023-07-19T18:44:47.237501712Z Modification: Change ping options
media_torrent    | 2023-07-19T18:44:47.240611758Z Modification: Update/set resolv-retry to 15 seconds
media_torrent    | 2023-07-19T18:44:47.242108284Z Modification: Change tls-crypt keyfile path
media_torrent    | 2023-07-19T18:44:47.243397144Z Modification: Set output verbosity to 3
media_torrent    | 2023-07-19T18:44:47.244973181Z Modification: Remap SIGUSR1 signal to SIGTERM, avoid OpenVPN restart loop
media_torrent    | 2023-07-19T18:44:47.249293597Z Modification: Updating status for config failure detection
media_torrent    | 2023-07-19T18:44:47.252253929Z Setting OpenVPN credentials...
media_torrent    | 2023-07-19T18:44:47.276623754Z adding route to local network 192.168.1.0/24 via 172.29.0.1 dev eth0
media_torrent    | 2023-07-19T18:44:47.279580323Z 2023-07-19 20:44:47 DEPRECATED OPTION: --cipher set to 'aes-128-cbc' but missing in --data-ciphers (AES-256-GCM:AES-128-GCM). Future OpenVPN version will ignore --cipher for cipher negotiations. Add 'aes-128-cbc' to --data-ciphers or change --cipher 'aes-128-cbc' to --data-ciphers-fallback 'aes-128-cbc' to silence this warning.
media_torrent    | 2023-07-19T18:44:47.280637751Z 2023-07-19 20:44:47 OpenVPN 2.5.5 x86_64-pc-linux-gnu [SSL (OpenSSL)] [LZO] [LZ4] [EPOLL] [PKCS11] [MH/PKTINFO] [AEAD] built on Jul 14 2022
media_torrent    | 2023-07-19T18:44:47.280912239Z 2023-07-19 20:44:47 library versions: OpenSSL 3.0.2 15 Mar 2022, LZO 2.10
media_torrent    | 2023-07-19T18:44:47.281167123Z 2023-07-19 20:44:47 NOTE: the current --script-security setting may allow this configuration to call user-defined scripts
media_torrent    | 2023-07-19T18:44:47.281827829Z 2023-07-19 20:44:47 CRL: loaded 1 CRLs from file -----BEGIN X509 CRL-----
media_torrent    | 2023-07-19T18:44:47.282051443Z MIICWDCCAUAwDQYJKoZIhvcNAQENBQAwgegxCzAJBgNVBAYTAlVTMQswCQYDVQQI
media_torrent    | 2023-07-19T18:44:47.282296500Z EwJDQTETMBEGA1UEBxMKTG9zQW5nZWxlczEgMB4GA1UEChMXUHJpdmF0ZSBJbnRl
media_torrent    | 2023-07-19T18:44:47.282506407Z cm5ldCBBY2Nlc3MxIDAeBgNVBAsTF1ByaXZhdGUgSW50ZXJuZXQgQWNjZXNzMSAw
media_torrent    | 2023-07-19T18:44:47.282710417Z HgYDVQQDExdQcml2YXRlIEludGVybmV0IEFjY2VzczEgMB4GA1UEKRMXUHJpdmF0
media_torrent    | 2023-07-19T18:44:47.282919282Z ZSBJbnRlcm5ldCBBY2Nlc3MxLzAtBgkqhkiG9w0BCQEWIHNlY3VyZUBwcml2YXRl
media_torrent    | 2023-07-19T18:44:47.283121020Z aW50ZXJuZXRhY2Nlc3MuY29tFw0xNjA3MDgxOTAwNDZaFw0zNjA3MDMxOTAwNDZa
media_torrent    | 2023-07-19T18:44:47.283323535Z MCYwEQIBARcMMTYwNzA4MTkwMDQ2MBECAQYXDDE2MDcwODE5MDA0NjANBgkqhkiG
media_torrent    | 2023-07-19T18:44:47.283572105Z 9w0BAQ0FAAOCAQEAQZo9X97ci8EcPYu/uK2HB152OZbeZCINmYyluLDOdcSvg6B5
media_torrent    | 2023-07-19T18:44:47.285472292Z jI+ffKN3laDvczsG6CxmY3jNyc79XVpEYUnq4rT3FfveW1+Ralf+Vf38HdpwB8EW
media_torrent    | 2023-07-19T18:44:47.286369379Z B4hZlQ205+21CALLvZvR8HcPxC9KEnev1mU46wkTiov0EKc+EdRxkj5yMgv0V2Re
media_torrent    | 2023-07-19T18:44:47.287261163Z ze7AP+NQ9ykvDScH4eYCsmufNpIjBLhpLE2cuZZXBLcPhuRzVoU3l7A9lvzG9mjA
media_torrent    | 2023-07-19T18:44:47.287531731Z 5YijHJGHNjlWFqyrn1CfYS6koa4TGEPngBoAziWRbDGdhEgJABHrpoaFYaL61zqy
media_torrent    | 2023-07-19T18:44:47.287777341Z MR6jC0K2ps9qyZAN74LEBedEfK7tBOzWMwr58A==
media_torrent    | 2023-07-19T18:44:47.288054093Z -----END X509 CRL-----
media_torrent    | 2023-07-19T18:44:47.288356730Z
media_torrent    | 2023-07-19T18:44:47.300887439Z 2023-07-19 20:44:47 TCP/UDP: Preserving recently used remote address: [AF_INET]143.244.41.182:1198
media_torrent    | 2023-07-19T18:44:47.301160035Z 2023-07-19 20:44:47 Socket Buffers: R=[212992->212992] S=[212992->212992]
media_torrent    | 2023-07-19T18:44:47.301371459Z 2023-07-19 20:44:47 UDP link local: (not bound)
media_torrent    | 2023-07-19T18:44:47.301621254Z 2023-07-19 20:44:47 UDP link remote: [AF_INET]143.244.41.182:1198
media_torrent    | 2023-07-19T18:44:47.319375051Z 2023-07-19 20:44:47 TLS: Initial packet from [AF_INET]143.244.41.182:1198, sid=762fb6ed d00f03db
media_torrent    | 2023-07-19T18:44:47.340815547Z 2023-07-19 20:44:47 VERIFY OK: depth=1, C=US, ST=CA, L=LosAngeles, O=Private Internet Access, OU=Private Internet Access, CN=Private Internet Access, name=Private Internet Access, emailAddress=secure@privateinternetaccess.com
media_torrent    | 2023-07-19T18:44:47.341187954Z 2023-07-19 20:44:47 VERIFY KU OK
media_torrent    | 2023-07-19T18:44:47.341519273Z 2023-07-19 20:44:47 Validating certificate extended key usage
media_torrent    | 2023-07-19T18:44:47.341862572Z 2023-07-19 20:44:47 ++ Certificate has EKU (str) TLS Web Server Authentication, expects TLS Web Server Authentication
media_torrent    | 2023-07-19T18:44:47.342190366Z 2023-07-19 20:44:47 VERIFY EKU OK
media_torrent    | 2023-07-19T18:44:47.342517461Z 2023-07-19 20:44:47 VERIFY OK: depth=0, C=US, ST=CA, L=LosAngeles, O=Private Internet Access, OU=Private Internet Access, CN=amsterdam404, name=amsterdam404
media_torrent    | 2023-07-19T18:44:47.470641161Z 2023-07-19 20:44:47 Control Channel: TLSv1.3, cipher TLSv1.3 TLS_AES_256_GCM_SHA384, peer certificate: 2048 bit RSA, signature: RSA-SHA512
media_torrent    | 2023-07-19T18:44:47.470871670Z 2023-07-19 20:44:47 [amsterdam404] Peer Connection Initiated with [AF_INET]143.244.41.182:1198
media_torrent    | 2023-07-19T18:44:47.498820804Z 2023-07-19 20:44:47 PUSH: Received control message: 'PUSH_REPLY,comp-lzo no,redirect-gateway def1,route-ipv6 2000::/3,dhcp-option DNS 10.0.0.243,route-gateway 10.26.112.1,topology subnet,ping 10,ping-restart 60,ifconfig 10.26.112.43 255.255.255.0,peer-id 5,cipher AES-128-GCM'
media_torrent    | 2023-07-19T18:44:47.499246324Z 2023-07-19 20:44:47 OPTIONS IMPORT: timers and/or timeouts modified
media_torrent    | 2023-07-19T18:44:47.499517049Z 2023-07-19 20:44:47 OPTIONS IMPORT: compression parms modified
media_torrent    | 2023-07-19T18:44:47.499824295Z 2023-07-19 20:44:47 OPTIONS IMPORT: --ifconfig/up options modified
media_torrent    | 2023-07-19T18:44:47.500079402Z 2023-07-19 20:44:47 OPTIONS IMPORT: route options modified
media_torrent    | 2023-07-19T18:44:47.500334973Z 2023-07-19 20:44:47 OPTIONS IMPORT: route-related options modified
media_torrent    | 2023-07-19T18:44:47.500601940Z 2023-07-19 20:44:47 OPTIONS IMPORT: --ip-win32 and/or --dhcp-option options modified
media_torrent    | 2023-07-19T18:44:47.500836923Z 2023-07-19 20:44:47 OPTIONS IMPORT: peer-id set
media_torrent    | 2023-07-19T18:44:47.501053281Z 2023-07-19 20:44:47 OPTIONS IMPORT: adjusting link_mtu to 1625
media_torrent    | 2023-07-19T18:44:47.501443718Z 2023-07-19 20:44:47 OPTIONS IMPORT: data channel crypto options modified
media_torrent    | 2023-07-19T18:44:47.505241220Z 2023-07-19 20:44:47 Data Channel: using negotiated cipher 'AES-128-GCM'
media_torrent    | 2023-07-19T18:44:47.505483815Z 2023-07-19 20:44:47 Outgoing Data Channel: Cipher 'AES-128-GCM' initialized with 128 bit key
media_torrent    | 2023-07-19T18:44:47.505760035Z 2023-07-19 20:44:47 Incoming Data Channel: Cipher 'AES-128-GCM' initialized with 128 bit key
media_torrent    | 2023-07-19T18:44:47.506008304Z 2023-07-19 20:44:47 net_route_v4_best_gw query: dst 0.0.0.0
media_torrent    | 2023-07-19T18:44:47.506210719Z 2023-07-19 20:44:47 net_route_v4_best_gw result: via 172.29.0.1 dev eth0
media_torrent    | 2023-07-19T18:44:47.506424433Z 2023-07-19 20:44:47 ROUTE_GATEWAY 172.29.0.1/255.255.0.0 IFACE=eth0 HWADDR=02:42:ac:1d:00:04
media_torrent    | 2023-07-19T18:44:47.506654200Z 2023-07-19 20:44:47 GDG6: remote_host_ipv6=n/a
media_torrent    | 2023-07-19T18:44:47.506914110Z 2023-07-19 20:44:47 net_route_v6_best_gw query: dst ::
media_torrent    | 2023-07-19T18:44:47.507199088Z 2023-07-19 20:44:47 sitnl_send: rtnl: generic error (-101): Network is unreachable
media_torrent    | 2023-07-19T18:44:47.507444470Z 2023-07-19 20:44:47 ROUTE6: default_gateway=UNDEF
media_torrent    | 2023-07-19T18:44:47.507702490Z 2023-07-19 20:44:47 TUN/TAP device tun0 opened
media_torrent    | 2023-07-19T18:44:47.507937303Z 2023-07-19 20:44:47 net_iface_mtu_set: mtu 1500 for tun0
media_torrent    | 2023-07-19T18:44:47.508143465Z 2023-07-19 20:44:47 net_iface_up: set tun0 up
media_torrent    | 2023-07-19T18:44:47.508372795Z 2023-07-19 20:44:47 net_addr_v4_add: 10.26.112.43/24 dev tun0
media_torrent    | 2023-07-19T18:44:47.508620767Z 2023-07-19 20:44:47 net_route_v4_add: 143.244.41.182/32 via 172.29.0.1 dev [NULL] table 0 metric -1
media_torrent    | 2023-07-19T18:44:47.508862662Z 2023-07-19 20:44:47 net_route_v4_add: 0.0.0.0/1 via 10.26.112.1 dev [NULL] table 0 metric -1
media_torrent    | 2023-07-19T18:44:47.511642109Z 2023-07-19 20:44:47 net_route_v4_add: 128.0.0.0/1 via 10.26.112.1 dev [NULL] table 0 metric -1
media_torrent    | 2023-07-19T18:44:47.513049845Z 2023-07-19 20:44:47 WARNING: OpenVPN was configured to add an IPv6 route. However, no IPv6 has been configured for tun0, therefore the route installation may fail or may not work as expected.
media_torrent    | 2023-07-19T18:44:47.513263284Z 2023-07-19 20:44:47 add_route_ipv6(2000::/3 -> :: metric -1) dev tun0
media_torrent    | 2023-07-19T18:44:47.513475736Z 2023-07-19 20:44:47 net_route_v6_add: 2000::/3 via :: dev tun0 table 0 metric -1
media_torrent    | 2023-07-19T18:44:47.513737146Z 2023-07-19 20:44:47 sitnl_send: rtnl: generic error (-13): Permission denied
media_torrent    | 2023-07-19T18:44:47.514013256Z 2023-07-19 20:44:47 ERROR: Linux IPv6 route can't be added
media_torrent    | 2023-07-19T18:44:47.514256825Z Up script executed with device=tun0 ifconfig_local=10.26.112.43
media_torrent    | 2023-07-19T18:44:47.514485299Z Updating TRANSMISSION_BIND_ADDRESS_IPV4 to the ip of tun0 : 10.26.112.43
media_torrent    | 2023-07-19T18:44:47.528651940Z Enforcing ownership on transmission directories
media_torrent    | 2023-07-19T18:44:47.529432071Z Applying permissions to transmission directories
media_torrent    | 2023-07-19T18:44:47.530177644Z Setting owner for transmission paths to 1000:1000
media_torrent    | 2023-07-19T18:44:47.530965254Z Setting permissions for download and incomplete directories
media_torrent    | 2023-07-19T18:44:47.554597859Z /etc/transmission/userSetup.sh: line 56: "002": syntax error: operand expected (error token is ""002"")
media_torrent    | 2023-07-19T18:44:47.554996931Z /etc/transmission/userSetup.sh: line 57: "002": syntax error: operand expected (error token is ""002"")
media_torrent    | 2023-07-19T18:44:47.555115050Z umask: 60
media_torrent    | 2023-07-19T18:44:47.555817698Z Directories:
media_torrent    | 2023-07-19T18:44:47.557203917Z chmod: invalid mode: ''
media_torrent    | 2023-07-19T18:44:47.558107921Z Files:
media_torrent    | 2023-07-19T18:44:47.560237580Z Try 'chmod --help' for more information.
media_torrent    | 2023-07-19T18:44:47.560551333Z chmod: invalid mode: ''
media_torrent    | 2023-07-19T18:44:47.560329260Z Setting permission for watch directory (775) and its files (664)
media_torrent    | 2023-07-19T18:44:47.560907320Z Try 'chmod --help' for more information.
media_torrent    | 2023-07-19T18:44:47.562466312Z
media_torrent    | 2023-07-19T18:44:47.562467365Z -------------------------------------
media_torrent    | 2023-07-19T18:44:47.562760573Z Transmission will run as
media_torrent    | 2023-07-19T18:44:47.563059227Z -------------------------------------
media_torrent    | 2023-07-19T18:44:47.563370583Z User name:   abc
media_torrent    | 2023-07-19T18:44:47.563665067Z User uid:    1000
media_torrent    | 2023-07-19T18:44:47.563976711Z User gid:    1000
media_torrent    | 2023-07-19T18:44:47.564299048Z -------------------------------------
media_torrent    | 2023-07-19T18:44:47.564562282Z
media_torrent    | 2023-07-19T18:44:47.564565282Z Updating Transmission settings.json with values from env variables
media_torrent    | 2023-07-19T18:44:47.583520378Z Attempting to use existing settings.json for Transmission
media_torrent    | 2023-07-19T18:44:47.584041629Z Successfully used existing settings.json /data/transmission-home/settings.json
media_torrent    | 2023-07-19T18:44:47.587789657Z Overriding bind-address-ipv4 because TRANSMISSION_BIND_ADDRESS_IPV4 is set to 10.26.112.43
media_torrent    | 2023-07-19T18:44:47.589069932Z Overriding download-dir because TRANSMISSION_DOWNLOAD_DIR is set to /downloads
media_torrent    | 2023-07-19T18:44:47.589342554Z Overriding idle-seeding-limit-enabled because TRANSMISSION_IDLE_SEEDING_LIMIT_ENABLED is set to false
media_torrent    | 2023-07-19T18:44:47.589607272Z Overriding incomplete-dir because TRANSMISSION_INCOMPLETE_DIR is set to /data/incomplete
media_torrent    | 2023-07-19T18:44:47.589890594Z Overriding ratio-limit because TRANSMISSION_RATIO_LIMIT is set to 1
media_torrent    | 2023-07-19T18:44:47.590197855Z Overriding ratio-limit-enabled because TRANSMISSION_RATIO_LIMIT_ENABLED is set to true
media_torrent    | 2023-07-19T18:44:47.590523853Z Overriding rpc-password because TRANSMISSION_RPC_PASSWORD is set to [REDACTED]
media_torrent    | 2023-07-19T18:44:47.590794207Z Overriding rpc-port because TRANSMISSION_RPC_PORT is set to 9091
media_torrent    | 2023-07-19T18:44:47.591063477Z Overriding rpc-username because TRANSMISSION_RPC_USERNAME is set to
media_torrent    | 2023-07-19T18:44:47.591330503Z Overriding speed-limit-up because TRANSMISSION_SPEED_LIMIT_UP is set to 1000
media_torrent    | 2023-07-19T18:44:47.591592079Z Overriding speed-limit-up-enabled because TRANSMISSION_SPEED_LIMIT_UP_ENABLED is set to true
media_torrent    | 2023-07-19T18:44:47.591871756Z Overriding watch-dir because TRANSMISSION_WATCH_DIR is set to /data/watch
media_torrent    | 2023-07-19T18:44:47.592158530Z sed'ing True to true
media_torrent    | 2023-07-19T18:44:47.592484047Z STARTING TRANSMISSION
media_torrent    | 2023-07-19T18:44:47.592826959Z Provider PIA has a script for automatic port forwarding. Will run it now.
media_torrent    | 2023-07-19T18:44:47.593197511Z If you want to disable this, set environment variable DISABLE_PORT_UPDATER=true
media_torrent    | 2023-07-19T18:44:47.596070919Z Transmission startup script complete.
media_torrent    | 2023-07-19T18:44:47.599019355Z 2023-07-19 20:44:47 WARNING: this configuration may cache passwords in memory -- use the auth-nocache option to prevent this
media_torrent    | 2023-07-19T18:44:47.599333531Z 2023-07-19 20:44:47 Initialization Sequence Completed
media_torrent    | 2023-07-19T18:44:52.597305298Z Running functions for token based port fowarding
media_torrent    | 2023-07-19T18:44:53.513901168Z transmission auth not required
media_torrent    | 2023-07-19T18:44:53.522823858Z transmission became responsive
media_torrent    | 2023-07-19T18:44:53.523834048Z      2     1%   398.5 MB  Unknown      0.0     0.0   0.00  Stopped      Black.Mirror.S06.2160p.NF.WEB-DL.DDP5.1.HDR.HEVC-NTb
media_torrent    | 2023-07-19T18:44:53.524065484Z Sum:             9.63 GB               0.0     0.0
media_torrent    | 2023-07-19T18:44:53.531177384Z setting transmission port to 31529
media_torrent    | 2023-07-19T18:44:53.568521996Z localhost:9091/transmission/rpc/ responded: success
media_torrent    | 2023-07-19T18:44:53.570041618Z Checking port...
media_torrent    | 2023-07-19T18:45:04.038418523Z Port is open: Yes
media_torrent    | 2023-07-19T18:45:04.041574919Z         SUCCESS
media_torrent    | 2023-07-19T18:45:04.045342877Z Port: 31529
media_torrent    | 2023-07-19T18:45:04.046880646Z #######################
media_torrent    | 2023-07-19T18:45:04.047349442Z Every 15 minutes, check port status

### HW/SW Environment

```markdown
- OS: Ubuntu 21.10
- Docker: Docker version 20.10.17, build 100c701
```

### Anything else?

![image](https://github.com/haugene/docker-transmission-openvpn/assets/2886499/c8f35da0-3546-4fa5-a30a-7afb168117c4)
![image](https://github.com/haugene/docker-transmission-openvpn/assets/2886499/a6862fc1-dafd-4f46-b68a-66c4b12c34b4)
![image](https://github.com/haugene/docker-transmission-openvpn/assets/2886499/7bd342a4-7afa-45d6-a5db-c08008f37680)
![image](https://github.com/haugene/docker-transmission-openvpn/assets/2886499/51fa70e7-b456-4614-a33e-113a621a27d9)
![image](https://github.com/haugene/docker-transmission-openvpn/assets/2886499/b6aba2a9-c40b-499c-ace0-71deaede4d2d)
![image](https://github.com/haugene/docker-transmission-openvpn/assets/2886499/88807405-9f7f-45f6-b6bb-daa9b2d3cea0)

**Comment by pkishino** [comment]
Container is behaving as it should, there is no container issue from what I can see. you could check the transmission issue tracker or with your provider, or possibly your tracker doesn’t support transmission 4.x yet..
Writing: tried everything
Is very unhelpful… 
Revert to an older release and check if it works
Try a different server
Try different trackers..

I’ll convert this to a discussion as it’s not really a container issue

**Comment by Zjaaspoer** [comment]
```
Writing: tried everything
Is very unhelpful…
```
fair enough, apologies

Indeed tried different trackers, different servers. The only thing I'm unable to switch is VPN provider (PIA right now) as I don't have any additional subscriptions.

Will also try older releases

**Reply by Zjaaspoer** [reply]
ok, just took a subscription on NordVPN, exact same problem. Also tried multiple trackers here, same problem (downloading is no issues, but peer can't seem to connect.

**Reply by Zjaaspoer** [reply]
downgraded to version 4, nothing changed.

**Reply by pkishino** [reply]
in that case your most likely culprit is your docker network..are you creating a network for this container or using defaults etc..

**Reply by Zjaaspoer** [reply]
Not an expert, but as far as I know I'm using the default network setup. The docker-compose.yaml is all there is to my setup. Also nothing specific on my overall Docker install

**Comment by rpgdev** [comment]
@Zjaaspoer did you ever find the culprit? I'm getting the same issue. Downloads fine but upload doesn't work and I've tried slow torrents with several leeches downloading that could leech from me but seemingly can't.

**Comment by Zjaaspoer** [comment]
@rpgdev no I moved away from this docker to a simple setup of
* Docker with Transmission
* Docker with VPN

Having the Docker with Transmission use the network of the Docker with the VPN. Much more control / visibility and works perfectly!

**Reply by rpgdev** [reply]
Interesting, will explore that! Any useful links for the docker with vpn setup that you can share?

**Reply by Zjaaspoer** [reply]
This is an abbreviated version of my docker-compose.yaml that should give you all the necessary info:

```
services:
  media_torrent:
    network_mode: service:media_vpn
    image: lscr.io/linuxserver/transmission:latest
  media_vpn:
    cap_add:
      - NET_ADMIN
    container_name: media_vpn
    environment:
      OPENVPN_PASSWORD: ***
      OPENVPN_USER: ***
      VPN_PORT_FORWARDING: 'on'
      VPN_SERVICE_PROVIDER: ***
      VPN_TYPE: openvpn
    image: qmcgaw/gluetun:latest
```
